### DISCUSSION TITLE
Will connect to macOS Monterey but not to the app

### DISCUSSION THREAD

**Discussion by ReasonableCause** [soundshed/soundshed-app#54] [Q&A] [answered]
Hi,

I can see and connect the amp from the Bluetooth menu, but the app continually scans and never finds the amp.

I have tried with the app 'pre-connected' in Bluetooth menu, and without. Neither helps. And I have factory reset the amp between tries.

Any suggestions?

macOS Monterey 12.3.1
2020 iMac (i7 not M1)
Soundshed v1.0.26

**Comment by webprofusion-chrisc** [comment, marked as answer]
Please try the latest version (v1.0.29) which uses a different method to connect to bluetooth and has a newer version of electron.
